pkg/datapath/linux/node_ids.go

```diff
@@ -92,6 +92,12 @@ func (n *linuxNodeHandler) allocateIDForNode(node *nodeTypes.Node) (uint16, erro
 		}
```

Comment:
I'm not able to parse the description for the first commit. I get what the commit's intent is, but might be worth rephrasing for others.

Reply:
I changed it, lmk if it looks better.